app/views/News.js

```diff
@@ -69,14 +70,16 @@ class NewsScreen extends Component {
             borderBottomLeftRadius: 12,
             borderBottomRightRadius: 12,
           }}
+          style={{ flex: 1, width: '100%' }}
```

Comment:
You're placing `width: 100%` on all children. It looks like you want `align-items: stretch` on the parent, rather than `width: 100%` on all children.

Reply:
these are changed now